src/pages/update-timetable-status-page/UpdateTimetableStatusPage.tsx

```diff
@@ -29,7 +29,7 @@ export const UpdateTimetableStatusPage = ({
               </li>
               <li>not adopted by the Local Authority</li>
             </ul>
-          </p>
+          </div>
```

Comment:
I'd been meaning to fix this to not have the `ul` inside the body element, like we do here: https://github.com/digital-land/local-plans-timetable/blob/54077f40c589e01a331c79af72d2278fb10b0bd5/src/pages/how-to-publish-page/HowToPublishPage.tsx#L17-L29

Maybe something to clean up now since we're here?

Reply:
done! :)